### ISSUE TITLE
[FEATURE] Provide variables on api call

### ISSUE THREAD

**Issue by AdamSGit** [FlowiseAI/Flowise#1079]
**Describe the feature you'd like**
With prompts node, you can use some `{variable}` that are replaced with the value of the « Format prompt values » node section. This json has an input node to provide the substitution. However, it seem that no mechanism allow to pass variables other than the `{{question}}` on querying the api.

Real word example using the translator flow that can be found in the marketplace section. The prompt system content is : 
`You are a helpful assistant that translates {input_language} to {output_language}.`

In this example, input and output langague are hardcoded in the « Format prompt values ». One would expect a way to controle thoses value, such as follow : 

```bash
curl http://localhost:3000/api/v1/prediction/uuid \
    -X POST \
    -d '{"question": "The string to be translated", "input_language": "English", "output_language": "French"}' \
    -H "Content-Type: application/json"
```

And the « Format prompt values » being : 
```json
{
  "input_language": "{{input_language}}",
  "output_language": "{{output_language}}",
}
```

**Comment by HenryHengZJ** [comment]
One alternative way is to use `overrideConfig` to override the whole prompt. For example:

```
overrideConfig: {
  systemMessagePrompt: You are a helpful assistant that translates English to French.
}
```

**Comment by AdamSGit** [comment]
Thanks for your answer. This is a workaround, sure, but it force the client to know the prompt to override it. When you think about it, if you modify the prompt in flowise, you also need to update it in the client's source code, which kind of invalidate the point of having the flow managment in flowise. Any other idea, or considering implementing that feature ?

**Comment by HenryHengZJ** [comment]
actually we can override the prompt values like so:
```
"overrideConfig": {
        "promptValues": {
            "input_language": "English",
            "output_language": "Italian"
        }
    }
```
Now, there was a bug that prevented it parsing correctly, the bugfix has been merged: https://github.com/FlowiseAI/Flowise/pull/1130

**Comment by AdamSGit** [comment]
Nice to hear. Thanks !

**Comment by HenryHengZJ** [comment]
Sure feel free to open a PR on the docs repo

**Comment by SirLucian** [comment]
Will such functionality be coming agents? for example, I want to use similar prompt templates with the Tool Agent

**Comment by umangkalavadiya** [comment]
I am facing the same issue. I will explain it. 
Suppose I want to create a story based on 2 variables in it. 
1. Title 2. Main character.

I have formatted prompt values and attached {{question}} to the title, and {{char}} to the Main character in the worker prompt.

I have enabled the override config and promptValues under the Worker.

This is my payload:
`{
    "question": "Heaven in the shell",
     "overrideConfig": {
        "promptValues":{
        "char": "Krishna the god"
        }
    }
}
`

I am getting some random story without the main character inclusion.
